service/waterfall.go

```diff
@@ -12,25 +13,21 @@ import (
 	"github.com/evergreen-ci/evergreen/model/build"
 	"github.com/evergreen-ci/evergreen/model/task"
 	"github.com/evergreen-ci/evergreen/model/version"
+	"github.com/evergreen-ci/gimlet"
 	"github.com/pkg/errors"
 )
 
 const (
-	// VersionItemsToCreate is the number of waterfall versions to create,
-	// including rolled-up ones.
-	VersionItemsToCreate = 5
-
-	// SkipQueryParam is the string field for the skip value in the URL
-	// (how many versions to skip).
-	SkipQueryParam = "skip"
-
-	InactiveStatus = "inactive"
+	InactiveStatus         = "inactive"
+	WaterfallPerPageLimit  = 5
+	WaterfallBVFilterParam = "bv_filter"
+	WaterfallSkipParam     = "skip"
```

Comment:
I think all of these values could be private, or at least it's not super obvious who the other users of these params are.